README.md

```diff
@@ -105,13 +105,23 @@ The following environment variables are available, and all of them are optional.
 
 ---
 
+#### `ENABLE_V6_EGRESS` (v1.13.0+)
+
+Type: Boolean as a String
+
+Default: `false`
+
+Specifies whether PODs in v4 cluster support IPv6 egress. If env is set to `true`, range fd00::ac:00/118 is reserved for PODs IPv6 egress.
```

Comment:
nit: let's surround `fd00::ac:00/118` in backticks, and change "reserved for PODs IPv6 egress" to "reserved for IPv6 egress"

Reply:
no problem.